samples/VanillaJSTestApp/index.html

```diff
@@ -25,7 +25,7 @@
     // initialize MSAL
     const msalConfig = {
         auth: {
-            clientId: "Enter_your_client_id",
+            clientId: "16b0c3aa-11cc-4afc-a44a-12f25e5872b6",
```

Comment:
Two separate `clientId` here, was this meant to be committed?

Reply:
Didn't mean to commit this, will remove it.